pybikes/whitebikes.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) Eduardo Mucelli Rezende Oliveira <edumucelli@gmail.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+import json
+
+from pybikes.base import BikeShareSystem, BikeShareStation
+from pybikes import utils
+
+
+class WhiteBikes(BikeShareSystem):
+
+    meta = {
+        'system': 'WhiteBikes'
+    }
+
+    def __init__(self, tag, meta, feed_url, bounding_box=None):
+        super(WhiteBikes, self).__init__(tag, meta)
+        self.feed_url = feed_url
+
+    def update(self, scraper=None):
+        scraper = scraper or utils.PyBikesScraper()
+
+        stations = []
+
+        data = json.loads(scraper.request(self.feed_url))
+
+        for station in data:
+
+            longitude = float(station['lon'])
+            latitude = float(station['lat'])
+
+            name = station['standName']
+            free = 0
```

Comment:
I recall we have some networks on we set `free` to None, and end up on the API as null. Long time ago we signaled no info on `free` with a -1, but in the end, I think `None` is the best.

Reply:
Yep, I was not sure whether it was 0, -1, or None.